rejuvenate.lua

```diff
@@ -1,43 +1,47 @@
--- make the selected dwarf 20 years old
+-- make the selected dwarf 20 years old, or the specified age with -age
 -- by vjek
 --@ module = true
 --[====[
 
 rejuvenate
 ==========
-Decreases the age of the selected dwarf to 20 years. Useful if valuable citizens
+Sets the age of the selected dwarf to 20 years old by default. Useful if valuable citizens
 are getting old.
 
 Arguments:
 
 - ``-all``: applies to all citizens
-- ``-force``: also applies to units under 20 years old. Useful if there are too many babies around...
+- ``-force``: also applies to units under 20 (or the age specified with -age) years old. Useful if there are too many babies around...
 - ``-dry-run``: only list units that would be changed; don't actually change ages
+- ``-age``: set to the specified age, rather than 20.
 
```

Comment:
this doc section is actually leftover from a doc migration we did last year. Could you delete this doc string and add your doc updates to `docs/rejuvenate.rst`?

Reply:
To confirm you want me to remove all documentation from the script?

Reply:
Donezo.